applications/asset_tracker_v2/src/modules/modem_module.c

```diff
@@ -702,6 +706,13 @@ static int setup(void)
 {
 	int err;
 
+#if IS_ENABLED(CONFIG_MEMFAULT) && IS_ENABLED(CONFIG_MEMFAULT_NCS_PROVISION_CERTIFICATES)
```

Comment:
The comment in overlay-memfault.conf says the app controls it, but is that elsewhere than here? `CONFIG_MEMFAULT_NCS_PROVISION_CERTIFICATES` is turned off in the overlay, so I don't think this code will run. Do the user need to run extra steps to provision the certificates another way?

Reply:
You're right, the last condition should be `!IS_ENABLED(CONFIG_MEMFAULT_NCS_PROVISION_CERTIFICATES)`. Fixed now